Add docs copy to examples (#1259)
mastra-ai/mastra: docs/src/pages/examples/llms/stream-text.mdx

# Stream Text

> This example shows you how to stream text from a language model. By using the `stream` method, you can capture the model's response in real-time, allowing you to handle the output incrementally.

Reviewer: ```suggestion
This example shows you how to stream text from a language model. The `stream` method allows you to stream the model's response in real-time to your user.
```
